torchtext/data/iterator.py

```diff
@@ -252,6 +252,28 @@ def create_batches(self):
                                 sort_within_batch=self.sort_within_batch)
 
 
+def generate_iterators(datasets, batch_sizes, **kwargs):
```

Comment:
Let's write model without batching first, because batching is a perf. optimization and we haven't measured it yet.